packages/start-plugin-core/tests/importProtection/utils.test.ts

```
@@ -1,21 +1,26 @@
 import { describe, expect, test } from 'vitest'
+import { getImportSources } from '../../src/import-protection/analysis'
 
 import {
   buildResolutionCandidates,
   buildSourceCandidates,
   canonicalizeResolvedId,
+  checkFileDenial,
+  dedupeViolationKey,
   dedupePatterns,
```

ESLint

[error] Member 'dedupePatterns' of the import declaration should be sorted alphabetically.

(sort-imports)